Assets/BemisCommsBehavior.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Linq;
+using JetBrains.Annotations;
+using UnityEngine;
+
+public class BemisCommsBehavior : MonoBehaviour
+{
+
+    private AudioSource bemisAudioSource;
+
+    [CanBeNull] private static AudioClip persistentAudioClip;
+    private static float? persistentAudioClipTime;
+
+    private void Start()
+    {
+        bemisAudioSource = GetComponent<AudioSource>();
+        if (persistentAudioClipTime.HasValue && persistentAudioClip != null)
+        {
+            bemisAudioSource.clip = persistentAudioClip;
+            bemisAudioSource.time = persistentAudioClipTime.Value;
+            bemisAudioSource.Play();
+            persistentAudioClip = null;
+            persistentAudioClipTime = null;
+        }
+    }
+
+    public void SavePersistentVoiceline()
```

Comment:
Rename this to `PlayPersistentVoiceline`

Reply:
but that's not what it does...

Reply:
What plays it?

Reply:
VoicelineManager